RFCs/RFC-0006: Redesign of Mina tokenomics.md

```diff
@@ -0,0 +1,79 @@
+- **Submitted on**: [26 Dec 2023]
+
+## Abstract
+
+Mina’s tokenomics, especially the high inflation brought by supercharged rewards, has been the concern of many community members, as evident by the number of posts on the [MIP channel of Mina Research](https://forums.minaprotocol.com/c/mips-mina-improvement-proposals/24). The upcoming hardfork is set to introduce zkApps (smart contracts), layer 2s, and many other exciting but complicated features to Mina, which also makes the original tokenomics design - a linear emission model - outdated. In this RFC idea, instead of proposing a tokenomics model directly, we outline the importance, opportunity and difficulty to design a successful model for Mina, and propose that Mina Foundation commission a thorough review and redesign of Mina’s tokenomics post hardfork, taking in account the critical aspects as outlined here.
+
+## Introduction
+
+**'Incentives are superpowers; set them carefully.'** - [Sam Altman](https://blog.samaltman.com/what-i-wish-someone-had-told-me).
+
+Tokenomics is as important as the consensus algorithm to a blockchain. On one hand, it contributes to the security by making attacking the chain more costly; on the other hand, it creates incentives for people to participate in the operation of the chain in various roles, from miners/block producers, to ecosystem builders, to end users.
+
+**Mina’s tokenomics is long outdated.** Mina’s current tokenomics was designed prior to the mainnet launch, and supercharged rewards (24% APR) for unlocked tokens was introduced to temporarily incentivize token holders to stake Mina. It worked at the time, but supercharged rewards were planned to be gradually wound down from 24% to 12% (equal to the APR for locked tokens) and eventually removed within 16 months of mainnet launch (see Section 6. Initial distribution Categories, 1d)Supercharged rewards of [this blog post](https://minaprotocol.com/blog/mina-token-distribution-and-supply)). The winding down of supercharged rewards did not get implemented, though, as the hardfork of smart contract was delayed by almost two years, and the inflation has been continuously running at >20% the whole time. This hyper inflation has been a concern for the community. Even though an MIP has been passed to remove supercharged rewards at the planned hardfork, which is expected in 2024, by the time the unplanned inflation will have run more than 15 months, and the theoretical inflation will still be 8.8% ([calculations by Gareth](https://forums.minaprotocol.com/t/mip-to-review-and-redesign-mina-tokenomics/6131/6): max of 7140 slots and 75% fill rate so 5355 max blocks. 5355 x 720 = 3.8556 million MINA increase an epoch. The current supply is 1.0724 billion, so that’s an increase of 0.00359517041% an epoch, annualized that’s 8.8%. If considering an active stake of 77.71%, the inflation rate would be ~8.0%). In comparison, the inflation rates of major L1 chains, for example, Ethereum, Solana and Cardano, are all < 7%.
```

Comment:
I think hyperinflation is a little dramatic ;) It's certainly higher than it needs to be. Also, staking rate != inflation rate. The inflation rate has historically been around 12%, which is actually very close to the original tokenomics (not really by design, but a combination of fixed coinbase and not all MINA staked).

Reply:
Hi Gareth, thanks for your comments. I've modified the wording to 'unplanned high inflation' referring to over-run SC rewards, and added a note on the historical inflation figure. Cheers.